0x000.md

```diff
@@ -0,0 +1,3 @@
+# 0x000 :octocat:
```

Comment:
Are you familiar with [Jekyll](https://help.github.com/articles/using-jekyll-with-pages/)? When it tries to generate the html from this markdown file, the content within `---` will be parsed out as [front matter](http://jekyllrb.com/docs/frontmatter/).